hibernate-core/src/test/java/org/hibernate/orm/test/query/hql/FunctionTests.java

```diff
@@ -1017,6 +1020,7 @@ public void testCastFunctionBinary(SessionFactoryScope scope) {
 	}
 
 	@Test
+	@SkipForDialect( dialectClass = AltibaseDialect.class, reason = "Altibase cast to char does not do truncatation")
```

Comment:
Shouldn't be necessary because the `@RequiresDialect` filter will ensure `AlitbaseDialect` is not even considered.

Reply:
I'm gonna remove this.